spec/dummy/config/application.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../boot', __FILE__)
+
+# Pick the frameworks you want:
+# require "active_record/railtie"
+require "action_controller/railtie"
+require "action_mailer/railtie"
+require "action_view/railtie"
+require "sprockets/railtie"
+# require "rails/test_unit/railtie"
+
+Bundler.require(*Rails.groups)
+require "europeana/blacklight"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.